docs/adr/adr_0001.adoc

```diff
@@ -116,81 +120,81 @@ Keep PersistenceProvider as they are and introduce new "MetaDataProvider" CRD wh
-* Introduces "New" Concept
-* Certificate Management for webhook services (`cert-manager` required by default?)
-* Scaling for systems with lots of load could be a problem
-* One service per namespace (multiple tenants) needed => results in many running active services which is ressource consuming
+* Introduces new running services which needs to be maintained and have uptime.
+* Code overhead / boilerplate (Can be mitigated by an SDK).
+* Debugging of individual _MetaDataProvider_ is harder than a single service which handles everything.
+* Introduces "`new`"cConcept.
+* Certificate management for webhook services (`cert-manager` required by default?).
+* Scaling for systems with lots of load could be a problem.
+* One service per namespace (multiple tenants) needed -> results in many running active services which is resource consuming.
 
 == Decision
 
-Regarding the Question 1 it seems that both solution approaches are resulting in the same execution model. We descided to implement solution approach 1 and unify both concepts into a more general concept with the name _"hook concept"_. Therefore we exchange the existing name `persistenceProvider` for phase 3 in the excecution model with a more general term `DataProcessing`:
+Regarding question 1 it seems that both solution approaches are resulting in the same execution model. We decided to implement solution approach 1 and unify both concepts into a more general concept with the name _hook concept_. Therefore we exchange the existing name _PersistenceProvider_ for phase 3 in the execution model with a more general term _DataProcessing_:
 
-```
-	┌──────────────────┐          ┌──────────────────┐          ┌──────────────────┐
-	│     Scanning     ├─────────▶│     Parsing      ├─────────▶│ DataProcessing   │
-	│    (Phase 1)     │          │    (Phase 2)     │          │    (Phase 3)     │
-	└──────────────────┘          └──────────────────┘          └──────────────────┘
-```
+// #30 I suggest the term "processing" instead of data processing/
+....
+┌──────────────────┐          ┌──────────────────┐          ┌──────────────────┐
+│     scanning     ├─────────▶│     parsing      ├─────────▶│  data processing │
+│    (Phase 1)     │          │    (Phase 2)     │          │    (Phase 3)     │
+└──────────────────┘          └──────────────────┘          └──────────────────┘
+....
 
-Regarding the Question 2 we decided to implement the solution approach 1 with a job-based approach (no active service componend needed).
-The Phase 3 `DataProcessing` will be therefore splitt into to seperate phases named `ReadAndWriteHooks (3.1)` and `ReadOnlyHooks (3.2)`
+Regarding question 2 we decided to implement the solution approach 1 with a job-based approach (no active service component needed). Therefore the phase 3 _data processing_ will be split into two separate phases named _ReadAndWriteHooks_ (3.1) and _ReadOnlyHooks_ (3.2)
+// #30 to what refers 3.1 and 3.2?
```

Comment:
The following image details the Phase 3 a bit more into two phases "Processing 3.1: ReadAndWriteHooks" and "Processing 3.2: ReadOnlyHooks" but the numbers are missing there :-/ Can you add them?

Reply:
Maybe we should use PlantUML:

- The AsciiArt does not render in all previous monospaced
- w/ images we can add numbers/titles and such easily with Asciidoc.

Reply:
Added it.